packages/python/plotly/plotly/io/_base_renderers.py

```diff
@@ -275,27 +273,13 @@ def activate(self):
                     )
                 )
 
-            if not self.requirejs:
-                raise ValueError("global_init is only supported with requirejs=True")
-
             if self.connected:
-                # Connected so we configure requirejs with the plotly CDN
                 script = """\
         <script type="text/javascript">
         {win_config}
         {mathjax_config}
-        if (typeof require !== 'undefined') {{
-        require.undef("plotly");
-        requirejs.config({{
-            paths: {{
-                'plotly': ['{plotly_cdn}']
-            }}
-        }});
-        require(['plotly'], function(Plotly) {{
-            window._Plotly = Plotly;
-        }});
-        }}
         </script>
+        <script type="module">import \"{plotly_cdn}\"</script>
```

Comment:
Wondering if we need to set `charset="utf-8"` here?

Reply:
What would be the advantage of that?

Reply:
I think we may skip it for now since out minified bundle does not have non ASCII characters.
But the advantage of setting `charset` is that if one wants to switch to the unminified bundle (e.f. for testing purposes) ; then they don't need to figure out these details as out unminified bundle may still have (or would possibly have) such characters. 
Here is an example of PRs that fixed unicode variable names in our for of d3 module: https://github.com/plotly/d3/pull/6/files